### DISCUSSION TITLE
Difference between policy types

### DISCUSSION THREAD

**Discussion by jannotti-glaucio** [eclipse-edc/Connector#4386] [General]
Hi guys !
What are the difference between the policy type possible values:
- set
- offer
- contract

I`m asking it because in the samples repo the policy is created with type = set, but in the contract's negotiation is used policy type = offer.
But if I use set on the policy creation and in the contract negotiations it works equals.

In which situation do I need to use each policy type and what behavior is expected for every policy type ?

Regargs,

**Comment by ndr-brt** [comment]
I suggest you to take a look at the [ODRL model documentation](https://www.w3.org/TR/odrl-model/#policy)
